src/_guides/language/formatting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Formatting code
+description: Use dartfmt to format your code, and follow Effective Dart guidelines for what dartfmt doesn't cover.
```

Comment:
nit: Lots of different line lenghts. Maybe auto-wrap everything at 80 cols? (especially given this is a page about automating formatting ;-)

Reply:
You sound like Bob! I prefer breaking on phrases, to minimize diffs caused by whitespace. It's like breaking after a semicolon... or whatever. :)